src/GoodsList.tsx

```diff
@@ -6,9 +6,13 @@ type Props = {
 };
 
 export const GoodsList: React.FC<Props> = ({ goods }) => (
-  <ul>
+  <ul className="Goods">
```

Comment:
```suggestion
  <ul className="goods">
```
Let's use kebab case for class names.